app/controllers/schedules_controller.rb

```diff
@@ -1,8 +1,12 @@
 class SchedulesController < ApplicationController
   protect_from_forgery with: :null_session
   before_action :respond_to_options
-  load_and_authorize_resource :conference, find_by: :short_title
+  skip_authorization_check
```

Comment:
you shouldn't skip the authorisation and handle it here. You should do it in the ability file, see: https://github.com/BelieveC/osem/blob/edb11d195dffac16597da6bf1f53d5678d208d46/app/models/ability.rb#L37